main.cpp

```diff
@@ -58,64 +57,124 @@ video: Chapter 2 - Part 3
 //2)
 void variableDeclarations()
 {
-    //example:
-    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
-    
-
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    bool isOn = 0;
+    bool isHigherThan5 = 1;
+    bool isOrange = 0;
```

Comment:
Never initialize boolean variables with `0` or `1`.  Always use `true` or `false`.  it is very clear what the intention is when you do that.
Any value OTHER THAN 0 is TRUE for booleans.   This is why you do not use numbers to initialize booleans.
- [ ] revise